src/decoder.c

```diff
@@ -385,3 +385,29 @@ int nanocbor_skip(nanocbor_value_t *it)
 {
     return _skip_limited(it, NANOCBOR_RECURSION_MAX);
 }
+
+bool nanocbor_get_key_tstr(nanocbor_value_t *start, const char *key,
+                           nanocbor_value_t *value)
+{
+    size_t len = strlen(key);
+    *value = *start;
+
+    while (!nanocbor_at_end(value)) {
+        const uint8_t *s;
+        size_t s_len;
```

Comment:
The linter complains about these variables not being initialized here:
```suggestion
        const uint8_t *s = NULL;
        size_t s_len = 0;
```
I know that they, with the current code, can never be used uninitialized, but clang-tidy considers it good style to always have an initialization value.